service/src/main/java/greencity/service/EcoNewsServiceImpl.java

```diff
@@ -103,7 +103,7 @@ public AddEcoNewsDtoResponse save(AddEcoNewsDtoRequest addEcoNewsDtoRequest,
         achievementCalculation
             .calculateAchievement(userVO, AchievementCategoryType.CREATE_NEWS, AchievementAction.ASSIGN);
         ratingCalculation.ratingCalculation(ratingPointsRepo.findByNameOrThrow("CREATE_NEWS"),
-            modelMapper.map(toSave, UserVO.class));
+            userVO);
```

Comment:
do you have tests for this method?
If you have please update them otherwise add tests